client/src/components/Chat/ChatPage/ChatPage.jsx

```diff
@@ -0,0 +1,33 @@
+import React from 'react';
+import MessagesContainer from '../MessagesContainer/MessagesContainer';
+import Input from '../Input/Input';
+import SideChat from '../SideChat/SideChat';
+import { useDispatch, useSelector } from 'react-redux';
+import { useParams } from 'react-router';
+import { setCurrentPerson } from '../../../reducers/chatReducer';
```

Comment:
to avoid such kind of paths I would recommend to setup aliases in webpack config